airbyte-webapp/src/pages/ConnectionPage/pages/ConnectionItemPage/components/ReplicationView.tsx

```diff
@@ -114,59 +99,62 @@ export const ReplicationView: React.FC<ReplicationViewProps> = ({ onAfterSaveSch
     if (activeUpdatingSchemaMode) {
       setActiveUpdatingSchemaMode(false);
     }
-
-    formikHelpers?.resetForm({ values });
```

Comment:
:information_source: The `onSubmit` function was never called with the `formikHelpers` parameters specified, so this should have never been called.

Reply:
It seems to work. This should all be handled by the call inside the `ConnectionForm`: https://github.com/airbytehq/airbyte/blob/tim/per-stream-flow/airbyte-webapp/src/views/Connection/ConnectionForm/ConnectionForm.tsx#L167